_posts/2019-06-25-CRANsearcher-package-discovery.md

```diff
@@ -0,0 +1,23 @@
+
+In 2017, the 10,000th R package published to CRAN and this became a hot topic in the R community.  It turns out we were not the only people struggling to navigate through all of the options.  This provided the last bit of motivation we needed to create our own solution.  Thus, the idea of `CRANsearcher` was born to fulfill an unmet need.
+
+We went about building `CRANsearcher` with the analysis workflow in mind.  The application, which allows for efficient navigation of tens of thousands of R packages, can be run directly from R.   With the click of a button in RStudio, `CRANsearcher` is launched; it loads the entire CRAN package database (in real time) into a table that is sortable and filterable by release date.  The user can perform multi-term search to discover packages through their names and descriptions.  Packages can be explored further through external links and installed with the click of a button.
+
+<img src="https://raw.githubusercontent.com/RhoInc/CRANsearcher/master/inst/image/gif/CRANsearcher_addin.gif"/>
+
+Although this tool immediately became an asset for internal R users, we were taken by surprise at how quickly it was embraced and even celebrated by the larger community.  We strongly believed this application could be useful to others, so we hosted its code on the Rho [GitHub site](https://github.com/RhoInc/CRANsearcher).  As a result, we've received great [feedback](https://twitter.com/juliasilge/status/866796243447463937), external [issues](https://github.com/RhoInc/CRANsearcher/issues?q=is%3Aissue+is%3Aclosed) for feature requests/bugs, and thoughtfully contributed [code](https://github.com/RhoInc/CRANsearcher/pulls?q=is%3Apr+is%3Aclosed+author%3Arpodcast). Sharing this code publicly has been a wonderful lesson in the power of community and collaboration, as well as innovation and transparency, all ideals we value in the Data Science group.  
```

Comment:
```suggestion
Although this tool immediately became an asset for internal R users, we were surprised by how quickly it was embraced by the larger community.  We strongly believed this application could be useful to others, so we hosted its code on the Rho [GitHub site](https://github.com/RhoInc/CRANsearcher).  To date, the package has been downloaded almost 10,000 times since it's release in June 2017. We've also received great [feedback](https://twitter.com/juliasilge/status/866796243447463937), external [issues](https://github.com/RhoInc/CRANsearcher/issues?q=is%3Aissue+is%3Aclosed) for feature requests/bugs, and thoughtfully contributed [code](https://github.com/RhoInc/CRANsearcher/pulls?q=is%3Apr+is%3Aclosed+author%3Arpodcast). Sharing this code publicly has been a wonderful lesson in the power of community and collaboration, as well as innovation and transparency, all ideals we value in the Data Science group.  
```